pkgs/applications/virtualization/hyperkit/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchFromGitHub
+, Hypervisor, vmnet, SystemConfiguration
+, xpc, libobjc, dtrace
+}:
+
+let
+  # Make sure to keep those in sync
```

Comment:
These are not in sync. Tag ```v0.20190201``` referes to ```64bbfbec66900334d91bb2ac32de60b5187cf6c9```.

Question, why not only using the tag itself?

Reply:
The tool's `--version` option includes both in its response.  [The Makefile uses git to determine them dynamically](https://github.com/moby/hyperkit/blob/master/Makefile#L1-L2), but you can see the prePatch phase below which removes that usage.

But we could just use `v${version}` for both and make it impossible for the version and rev to fall out of sync.

Reply:
Actually, I'm wrong, compilation does require a CFLAG `-DVERSION_SHA1`, but I can't tell what it's used for.
```
% result/bin/hyperkit -v
hyperkit: v0.20190201

Homepage: https://github.com/docker/hyperkit
License: BSD
```

Reply:
I've updated the PR to use the tag for every version and "sha" string.

Reply:
I was wrong, that won't work, [there's a runtime error if the VERSION_SHA1 is not the right length](https://github.com/moby/hyperkit/blob/3e954c1068f2ebea39cbc0119fe4d0f31f015f3a/src/lib/pci_virtio_net_vpnkit.c#L324)